source/features/preview-hidden-comments.tsx

```diff
@@ -29,10 +29,11 @@ function init(): void {
 		}
 
 		// Hidden review comments that have been preloaded have their header text wrapped in an extra <div>
-		const headerTextWrapper = header.tagName === 'H3' ? select(':scope > .d-inline-block:last-child', header) : undefined;
+		// Comments that have just been hidden already have a `.Details-content--open` element #4983
+		const headerText = select(':scope > :is(.d-inline-block:last-child, .Details-content--open)', header) ?? header.firstChild;
 		header.append(
-			<span className="Details-content--open">{headerTextWrapper ?? header.firstChild}</span>,
-			<span className="Details-content--closed">{`${upperCaseFirst(reason)} — ${commentText.slice(0, 100)}`}</span>,
+			<span className="Details-content--open">{headerText}</span>,
```

Comment:
Test comment

Reply:
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Is es profecto tu. Non enim solum Torquatus dixit quid sentiret, sed etiam cur. Primum cur ista res digna odio est, nisi quod est turpis? Hanc ergo intuens debet institutum illud quasi signum absolvere. Obsecro, inquit, Torquate, haec dicit Epicurus? Duo Reges: constructio interrete. Quod quidem iam fit etiam in Academia. Duarum enim vitarum nobis erunt instituta capienda.